changed-images-to-hyperlinks

calderEDJ opened the pull request:
Removed images of collaborator and funder logos, added hyperlinks

calderEDJ committed:
changed-images-to-hyperlinks

Removed images of collaborator and funder logos, added hyperlinks

Frijol commented on content/about/about-content.md:
```diff
@@ -28,8 +28,12 @@ A lot of the work in EEW stems from EDGI’s work around “Environmental Data J
 
 ## Collaborators
-![EEW's collaborators: Stony Brook University Geospatial Center, Teaching Learning Lab, Sunrise Boston](./collabs.png)
+<a href="https://www.stonybrook.edu/commcms/gss/" target=_blank >Geospatial Center at Stony Brook University</a>
```
In Markdown, you have to hit enter twice to get things on different lines – currently these will all render on one line (you can see by clicking 
![Screen Shot 2020-10-02 at 1 39 11 PM](https://user-images.githubusercontent.com/454690/94967805-aa235a00-04b4-11eb-8d11-2c38fd4fcd8e.png)
in the "Files changed" tab)

To fix: either put an extra newline between each line, or use `*` or `-` to turn it into a bulleted list

Frijol commented on content/about/about-content.md:
```diff
@@ -28,8 +28,12 @@ A lot of the work in EEW stems from EDGI’s work around “Environmental Data J
 ## Collaborators
-![EEW's collaborators: Stony Brook University Geospatial Center, Teaching Learning Lab, Sunrise Boston](./collabs.png)
+<a href="https://www.stonybrook.edu/commcms/gss/" target=_blank >Geospatial Center at Stony Brook University</a>
+<a href="http://apps.tlt.stonybrook.edu/" target=_blank >Teaching Learning Lab at Stony Brook University</a>
```
nice! I'm impressed you picked up my html formatting with `target=_blank` – this extra argument when added to an `a href` tells the browser to open the link in a new tab.

Frijol commented on content/about/about-content.md:
```diff
@@ -28,8 +28,12 @@ A lot of the work in EEW stems from EDGI’s work around “Environmental Data J
 ## Funding
-![EEW's funders: The David and Lucile Packard Foundation, Doris Duke Charitable Foundation, Northeastern Unicersity Lab for Texts, Maps, and Networks](./funders.png)
+<a href="https://www.packard.org/" target=_blank >The David and Lucile Packard Foundation</a>
+<a href="https://www.ddcf.org/" target=_blank >Doris Duke Charitable Foundation</a>
```
Should there be agreement "The Packard"/"The Doris" or "Packard"/"Doris"? Or is this how they list themselves?

Frijol reviewed (changes requested):
Looks pretty good, couple of fixes!

calderEDJ committed:
fixed-formatting-issues

Fixed the collaborators and funding lists (gave them bullets) and added "The" in front of Doris Duke

Frijol commented:
This looks great, merging!

![Screen Shot 2020-10-06 at 8 56 01 AM](https://user-images.githubusercontent.com/454690/95226424-c4f61700-07b1-11eb-8245-de6bb938b5c0.png)

Just occurred to me that "Funding" should perhaps change to "Funders" in order to better match "Participants" and "Collaborators" – but, you can make that in a separate PR if you agree
